apps/explorer/src/pages/epochs/stats/EpochProgress.tsx

```diff
@@ -25,15 +25,31 @@ export function EpochProgress({
 }: EpochProgressProps) {
     const { progress, label } = useEpochProgress();
 
+    const elapsedTime =
+        !inProgress && start && end ? getElapsedTime(start, end) : undefined;
```

Comment:
Why don't we show this for in-progress epochs?

Reply:
Yeah I imagined we could use the current date but I think because the progress bar exists this doesn't make sense